content/20.pharmaceuticals.md

```diff
@@ -358,8 +363,17 @@ Six leads were identified, and _in vitro_ analysis revealed that ebselen, an org
 Ebselen is an organoselenium compound with anti-inflammatory and antioxidant properties [@doi:10.1007/s11033-014-3417-x]; however, it is likely to be a promiscuous binder, which could diminish its therapeutic potential [@doi:10.1038/s41586-020-2223-y].
 While there is clear computational and _in vitro_ support for ebselen's potential as a COVID-19 therapeutic, results from clinical trials are not yet available for this compound.
 
-<!--To Do: add paragraph about phospholipidosis story, which is screening-based and has more established results, and ask James Fraser to review. He suggested this paper: https://www.biorxiv.org/content/10.1101/2021.03.23.436648v1
-To Do: from @rlordan: Autopsy case study wonders whether HCQ caused renal phospholipidosis - https://pubmed.ncbi.nlm.nih.gov/33274661/-->
+Another candidate has emerged from a broader assessment of work in computational prediction of candidate drugs for repurposing against COVID-19.
+A large number of efforts have sought to collect data about interactions between drugs and SARS-CoV-2 and about the genomic response to SARS-CoV-2 exposure.
+These resources were integrated into the COVID-19 Drug and Gene Set Library, which at the time of its publication contained 1,620 drugs sourced from 173 experimental and computational drug sets and 18,676 human genes sourced from 444 gene sets [@doi:10.1016/j.patter.2020.100090].
+This study found that several drugs had been identified as candidates across multiple independent analyses, including several that had received significant attention as potential therapeutics for COVID-19 (e.g., CQ/HCQ and remdesivir) [@doi:10.1016/j.patter.2020.100090].
+A second study expanded on these results by identifying that cationic amphiphilicity was a shared property among many of the candidate drugs identified through both computational and phenotypic screens, suggesting it as a potential cause of a compound's proposed antiviral activity [@doi:10.1101/2021.03.23.436648].
+Cationic amphiphilic drugs are known to initiate phospholipidosis, which is when phospholipids accumulate in the lysosome [@doi:10.1515/hsz-2019-0270]; indeed, one case study identified drug-induced phospholipidosis as the cause of death for a COVID-19 patient treated with HCQ [@doi:10.1080/01913123.2020.1850966].
```

Comment:
Maybe switch the order of this sentence and the next so that we continue to talk about the potential benefit of phospholipidosis before mentioning the negative case.

Reply:
I'm going to drop the "HCQ is so good at phosopholipidizing that it can kill you" but we might want to add this to the appendix!